webonary-cloud-api/lambda/db.ts

```diff
@@ -63,8 +63,122 @@ export interface DbFindParameters {
 
 export const DB_NAME = process.env.DB_NAME as string;
 
+export const DB_MAX_DOCUMENTS_PER_CALL = 100;
+export const DB_MAX_UPDATES_PER_CALL = 50;
+
 // Vietnamese seem to have the most Latin diacritics, so use this for case and diacritic insensitive search
-export const DB_COLLATION_INSENSITIVE = { collation: { locale: 'vi', strength: 1 } };
+export const DB_COLLATION_LOCALE_DEFAULT = 'vi';
```

Comment:
I think we've lost the descriptive variable name by this change.

Reply:
// TODO: Vietnamese seems to have the most Latin diacritics, so use this for case and diacritic insensitive search
// We might have to do RegExp searches instead to get more accurate insensitive searches
export const DB_COLLATION_LOCALE_DEFAULT_FOR_INSENSITIVITY = 'vi';
export const DB_COLLATION_STRENGTH_FOR_INSENSITIVITY = 1;